frontend/src/modules/taxons/actions.js

```diff
@@ -12,6 +14,24 @@ export const get = id => async (dispatch, getState) => {
     return await dispatch(networkAction(GET, api.taxons.get(id), { meta: { id } }));
 };
 
+export const add = taxon => async (dispatch, getState) => {
+    if (getState().taxons.isFetching)
+        return;
+
+    return await dispatch(networkAction(
+        ADD, api.taxons.add(taxon), { meta: { ignoreError: 'APIError' } }
+    ));
+};
+
+export const update = (id, taxon) => async (dispatch, getState) => {
+    if (getState().taxons.itemsByID[id].isFetching)
```

Comment:
I don't think this is necessary either. We should be able to call the backend to update even if the frontend is busy fetching this taxon. Did this fix a bug you ran into?